### PR TITLE
Initial Account and League API Endpoints

### PR DESCRIPTION
**Pull request by MichaelCduBois**
Based on our discussion Tuesday, this PR updates the POST API Endpoints to send and receive the expected data.

### CODE REVIEW

**Comment by MichaelCduBois** [comment]
@kylekrny please review.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The Leagueify server project contains core infrastructure (API, database, etc.) for the "backend" of all Leagueify applications.
 
-The server is written in TypeScript and uses [Prisma](https://www.prisma.io/) to manage the database schema and migrations.
+The server is written in TypeScript and uses [Mongoose](https://mongoosejs.com/) with [MongoDB](https://www.mongodb.com/) to manage the database schema.
```
Comment (kylekrny) [inline README.md:5]: Question: Why the change to Mongoose / MongoDB?

Comment (MichaelCduBois) [reply README.md:5]: Bun currently does not work with Prisma, here is a quote from their [docs](https://bun.sh/guides/ecosystem/prisma).

> Note — At the moment Prisma needs Node.js to be installed to run certain generation code. Make sure Node.js is installed in the environment where you're running bunx prisma commands.

NodeJS is not installed within the Bun Docker Image. I was not able to get Prisma working.

File: api/leagues/tests/leagues.test.ts [hunk 2]
```diff
@@ -1,24 +1,183 @@
...
+        name: "Youth Division",
+        open: false,
+        minAge: 5,
+        maxAge: 10,
+      },
+      {
+        name: "Adult Division",
+        open: false,
+        minAge: 18,
+        maxAge: 100,
```
Comment (kylekrny) [inline api/leagues/tests/leagues.test.ts:86]: Seems discriminatory against 101 year old hockey players. #stopageism

**Review by kylekrny — APPROVED** [review]
Looks great. Only last question what port is the documentation on?

**Comment by MichaelCduBois** [comment]
> Looks great. Only last question what port is the documentation on?

: 8888
